ios/FluentUI/Button/Button.swift

```diff
@@ -321,7 +343,7 @@ open class Button: UIButton {
         layer.borderWidth = style.hasBorders ? Constants.borderWidth : 0
 
         if !isUsingCustomContentEdgeInsets {
-            contentEdgeInsets = style.contentEdgeInsets
+            edgeInsets = style.contentEdgeInsets
```

Comment:
@harrieshin Just trying to understand your previous comment, does that mean we should be triggering config here as well?

Reply:
Setting edgeInsets will trigger the didSet of edgeInsets, so I think just adding it there should be fine.